middleware/browse/browse_test.go

```diff
@@ -1,14 +1,16 @@
 package browse
 
 import (
+        "encoding/json"
+	"github.com/mholt/caddy/middleware"
```

Comment:
Yah I figured it was something like that since normally your PRs are formatted pretty good :) No problem.